### DISCUSSION TITLE
Self-assemble of molecules

### DISCUSSION THREAD

**Discussion by junzhoux** [ricalessandri/Martini3-small-molecules#17] [Q&A] [answered]
Hi @ricalessandri, is it possible to optimize the new Martini 3 parameters for better modelling the self-assembling of molecules, like surfactants or lipids?  I try to parametrize several surfactant models followed the tutorial, but fails to reproduce the aggregating number of micelles.

**Comment by ricalessandri** [comment]
Hi @junzhoux , thanks for reaching out. Probably it is possible, yes. But what do you exactly mean by the "aggregating number of micelles"? Do you mean the "average number of surfactant monomers in a (spherical) micelle"? And how was your reference data obtained (experimental, another CG model, etc.)?

**Comment by junzhoux** [comment]
Hi @ricalessandri, I modelled a binary surfactants system (SLES/CAPB mixture), which will generate wormlike micelles at a mixing ratio of 2:1 (SLES:CAPB). For my simulation, however, I can only observe spherical micelles after 5 us. I also tried another popular wormlike micellar system (CTAB+NaSal), and I do observe long polymer-like micelles. I am just wondering if the interactions between polar head groups of surfactant are strong and they quick aggregate into small spherical micelles. If I scale down the interactions between beads in head part (P, Q) and tail part (C1), some rod micelles occurs.

Meanwhile, I also adopt another set of parameters (DPD) in LAMMPS for SLES/CAPB system (https://doi.org/10.1021/acs.langmuir.0c02210), the wormlike micelles are observed. But the parameterization process for new molecules use DPD method is a bit tricky compared Martini force field, I prefer using later for my simulation. 

Here is my mapping scheme for these two molecules (https://github.com/junzhoux/surfactant-model/blob/5e78cacde884ea4cccebe60ce46db5f291636b0b/mapping.png), please have a look.

**Comment by ricalessandri** [comment, marked as answer]
Hi @junzhoux , thanks for the detailed info. So, I would map SLES in this way:
https://github.com/ricalessandri/Martini3-mappings/blob/main/SLES-2EO.pdf
this seems more appropriate as it maximizes the 4-to-1 mappings and doesn't leave any atom out of the mapping. I don't see the reason for that SC3 bead in the mapping (and if you count the non-hydrogen atoms, you're not accounting for all of them; namely, neglecting the SO4 group, you have 18 atoms, and hence 3 x R-bead + 2 x S-bead (3x4+2x3=18) are needed). 
The proposed mapping may also have the effect that you want, i.e., effectively scaling down the interactions between the heads and the tail as you'll have a more hydrophobic tail (SC3 is quite a bit more hydrophilic than C1). Or, in other words,, the tail-tail interactions will be stronger and hence "compete more". Probably worth a try?

**Comment by ricalessandri** [comment]
Hi @junzhoux, I was wondering if you tried the different mapping and how things turned out.

**Comment by ricalessandri** [comment]
Hi @junzhoux, I will move this to Discussions.
